tokio-console/src/view/mod.rs

```diff
@@ -69,6 +69,7 @@ pub(crate) enum UpdateKind {
     SelectTask(u64),
     /// The TaskView is exited
     ExitTaskView,
+    #[allow(dead_code)]
```

Comment:
We might use this in the future. So I just allow the dead code check. Please let me know if we want to remove it.

Reply:
I think we used the task ID to monitor task details, but we don't need to use it for resources.